docs/DeveloperGuide.md

```diff
@@ -157,6 +157,69 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### addEvent feature
```

Comment:
Rather than the command itself, perhaps something like Adding Events would be clearer as to the exact feature we are trying to implement.